public/docs/ts/latest/glossary.jade

```diff
@@ -391,7 +381,7 @@ a#H
     Data values flow *into* this property from the data source identified
     in the template expression to the right of the equal sign.
 
-    See the [Template Syntax](!{docsLatest}/guide/template-syntax.html#inputs-outputs) chapter.
+    See the [Template Syntax](!{docsLatest}/guide/template-syntax.html#inputs-outputs) page.
```

Comment:
Went through all the links and clarified. Found a few that went to the wrong places and corrected. If you see one I didn't catch, please let me know and I'll fix ASAP.